domain/src/test/java/org/oppia/android/domain/classify/rules/textinput/TextInputContainsRuleClassifierProviderTest.kt

```diff
@@ -24,6 +24,27 @@ import kotlin.test.fail
 @Config(manifest = Config.NONE)
 class TextInputContainsRuleClassifierProviderTest {
 
+  private val STRING_VALUE_WILL_BREAK =
```

Comment:
Can we have some more understandable naming here? 
As per from the other file, we have naming like `STRING_VALUE_TEST_UPPERCASE ` which says, this will be testing an uppercase string value.

Reply:
Should I name it `STRING_VALUE_THIS_IS_A_TEST_I_WILL_BREAK` **OR** `STRING_VALUE_WILL_BREAK_MESSAGE`

Reply:
> > Thanks @ARJUPTA Left few comments.
> > Also, put a screenshot in the PR description showing all the test cases are passing.
> 
> @anandwana001 Can you direct me on how to run the test of another module?

Head over to our wiki page for testing - https://github.com/oppia/oppia-android/wiki/Oppia-Android-Testing

Reply:
Let's see what the test is trying to do, and name it accordingly. 
I don't think there is something related to break, although we have 'break' word in our string, that doesn't relate to anything breaking. 
I think this is related to the different values to test value match or not, check another test file in the package on a similar test what's the naming there.

Reply:
Unfortunately, there is no String in TextInput package with the same value

Reply:
Check `TextInputFuzzyEqualsRuleClassifierProviderTest.kt`

Reply:
check `TextInputFuzzyEqualsRuleClassifierProviderTest.kt`

Reply:
### **The string values there are -**
  private val STRING_ANSWER_UPPERCASE = createString(value = "TEST")
  private val STRING_INPUT_UPPERCASE = createString(value = "TEST")
  private val STRING_INPUT_LOWERCASE = createString(value = "test")
  private val STRING_ANSWER_LOWERCASE = createString(value = "test")
  private val STRING_ANSWER_DIFF_LOWERCASE = createString(value = "diff")
  private val STRING_INPUT_DIFF_LOWERCASE = createString(value = "diff")
  private val STRING_INPUT_DIFF_UPPERCASE = createString(value = "DIFF")
  private val STRING_FUZZY_INPUT = createString(value = "This Is a TesT")
  private val STRING_ANSWER = createString(value = "this is a test")
  private val STRING_WITH_WHITESPACE = createString(value = "  test   ")
**What are you suggesting to follow**